main.go

```diff
@@ -207,6 +208,11 @@ func main() {
 		replicationFactor int
 
 		nodes             string
+		caCertFile		  string
```

Comment:
Tabs seem to be mixed with spaces in this line. Please use spaces between `caCertFile` and `string`.

Reply:
fixed